src/index.html

```diff
@@ -22,6 +22,391 @@
+          alt="desktop"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__description">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="desktop"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__description">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="desktop"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__description">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
```

Comment:
The content within these `<div class="card">` elements is repeated multiple times. Consider updating each card with unique content to avoid redundancy and to better reflect the intended design or functionality.